src/Servers/IIS/test/IISIntegration.FunctionalTests/OutOfProcess/HttpsTest.cs

```diff
@@ -25,17 +25,15 @@ public HttpsTest(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Theory(Skip = "Full framework web.config generation is currently incorrect. See: https://github.com/aspnet/websdk/pull/322")]
+        [Theory]
         [InlineData("V1")]
```

Comment:
Theories with only one data point can be simplified to `Fact`...unless you're trying to avoid merge conflicts with 2.2 and master, in which case it's not a big deal.